registry/storage/driver/middleware/redirect/middleware.go

```diff
@@ -0,0 +1,47 @@
+package middleware
+
+import (
+	"fmt"
```

Comment:
Hmm...I tried `gofmt -s middleware.go` as the `fmt` taget in the `Makefile` specifies, but it doesn't seem to order it. I have go1.6.1 darwin/amd64 as a development environment. I'll order it by hand and commit.